src/config/gossip.conf

```diff
@@ -0,0 +1,12 @@
+; example gossip configuration file
```

Comment:
@svspire Is the intention to have a single global configuration committed to our source repository?  I think you would want these files to by examples, rather than universal configuration.

Reply:
It's all examples right now. Remove the whole config subdirectory if you wish. I hadn't intended to include it anyway.